vuepress/docs/next/docs/ecr/ecr-bundle-details.md

```diff
@@ -25,71 +31,71 @@ The bundle descriptor file aggregates all included components and has the follow
 
 > **Warning**
 >
-> Remember the file needs to be named `descriptor.yaml`.
+> Remember, the file must be named `descriptor.yaml`.
 
 **descriptor.yaml.**
 
     code: survey_bundle # The bundle ID
     description: This is the survey bundle # The description of the bundle
 
-    components: # All components will be here
+    components: # All components are listed here.
 
-      # Optional. Use if the component requires a deployment
+      # Optional. Use if the component requires deployment
       plugins:
         - folder/you/want/my_plugin_descriptor.yaml
         - folder/you/want/another_plugin_descriptor.yaml
 
-      # To create widgets you will need to add references to the descriptor file's
+      # To create Widgets, add references to the descriptor file's 
```

Comment:
Question, does it seem like these statements are redundant? I mean each of the "create X" lines, since the following line indicates the component type. The only interesting bit (which could be its own comment ahead of the YAML) is that the Page Template feature is pageModels, and Content Template is contentModels, in the yaml.

Reply:
absolutely redundant. i know you mentioned this in the meeting, as well. i was just working with what's there instead of putting energy into restructuring, but removing this language in favor of a short initial comment is probably just as much work! i'm also sensitive to the writing style (found often in manuals) where an instruction is repeated, so the comments do seem familiar/valid, but i agree they can be omitted

Reply:
Yeah, and I explicitly said don't worry about the yaml code - and then it completely derailed me. :)